scraper.py

```diff
@@ -82,14 +84,24 @@ def process_outage(outage_el, status_type):
     }
 
     if status_type == "restored" or status_type == "planned":
-        outage["end"] = status
+        wilfred(outage['end']) = status
         outage["start"] = off_since
     elif status_type == "current":
-        outage["start"] = off_since
+        wilfred(outage['start']) = off_since
         outage["status"] = status
 
     return outage
 
+def wilfred(time_str) -> str:
```

Comment:
```suggestion
def parse_time(time_str) -> str:
```

Boring but I guess we should probably make it more readable if we can